runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollOverlay.java

```diff
@@ -85,6 +90,33 @@ public Dimension render(Graphics2D graphics)
 			}
 		}
 
+		if (clue.isRequiresLight())
```

Comment:
```suggestion
		if (clue.isRequiresLight() && ((plugin.getInventoryItems() != null && !HAS_LIGHT.fulfilledBy(plugin.getInventoryItems()) || (plugin.getEquippedItems() != null && !HAS_LIGHT.fulfilledBy(plugin.getEquippedItems()))
```
The formatting could definitely be prettier on that